.github/workflows/ci.yml

```diff
@@ -1,35 +1,41 @@
 name: ci
 
-# Controls when the workflow will run
 on:
-
-  # Trigger the workflow on all pushes, except on tag creation
+  # Trigger the workflow on push to all branches, except master, develop and on tag creation
   push:
     branches:
-    - '**'
+      - '**'
```

Comment:
@figi44 could we get rid of the double CI trigger please? We can set `branches` to only `master` and `develop`, keep the `pull_request` trigger, then the CI will only trigger once when a PR is opened and pushed to.